.github/workflows/testing.yml

```diff
@@ -8,11 +8,16 @@ env:
 
 jobs:
   tests:
-    name: "Python ${{ matrix.python-version }}"
-    runs-on: "ubuntu-latest"
+    name: "Python ${{ matrix.python-version }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
Any particular reason why dont want to `fail-fast`?

Reply:
Not really, will remove it :)